### ISSUE TITLE
Add module-info.java

### ISSUE THREAD

**Issue by wdonne** [schibsted/jslt#159]
When a package that has a module-info.java uses JSLT, then it must add something for JSLT. Intellij proposes to add "requires jslt;" and then it compiles. However, with "mvn compile" you get this warning:

[WARNING] * Required filename-based automodules detected: [jslt-0.1.10.jar]. Please don't publish this project to a public artifact repository! *

Adding a module-info.java file to the JSLT project would solve this. The package can still remain compiled for Java 8 with the plugin https://maven.apache.org/plugins/maven-compiler-plugin/examples/module-info.html.

**Comment by larsga** [comment]
We probably should do this, but it's a bit tricky while also compiling to Java 8.

https://old.reddit.com/r/java/comments/nhukbq/i_wrote_jmail_a_lightweight_java_library_for/gyy9uba/

**Comment by wdonne** [comment]
Indeed, it requires some wieldy configuration in Maven. This is an example: https://github.com/wdonne/pincette-rs/blob/master/pom.xml.
